package.json

```diff
@@ -55,6 +56,7 @@
     "jest": "^21.2.1",
     "node-fetch": "^1.7.3",
     "prettier": "^1.13.4",
+    "rifraf": "^2.0.3",
```

Comment:
I think this should be `rimraf`?

Reply:
Yeah, true that, must have misstyped when `npm install` ing it. THX for pointing that out.